src/pages/broadcaster-sdk-ios/getting-started.mdx

```diff
@@ -26,3 +28,5 @@ If you have questions, please [contact us](https://video.ibm.com/enterprise-vide
 The IBM Video Streaming Broadcaster SDK for iOS requires iOS 9.0 or above.
 The IBM Video Streaming Broadcaster SDK for macOS requires macOS 10.11 or above.
 We recommend using the most recent version of Xcode.
+
+[contact us]: https://video.ibm.com/enterprise-video/contact
```

Comment:
Due to this line being added, we can alter line 24:
From this: `If you have questions, please [contact us](https://video.ibm.com/enterprise-video/contact).`
to this: `If you have questions, please [contact us].`

Because the contact us link is already defined.

Reply:
done